PKSim-DataBase/DB.md

```diff
@@ -1,5 +1,28 @@
 # Introduction
-PK-Sim uses an [SQLite](https://www.sqlite.org/index.html) database to store species, population etc. In this part of the documentation, you can find a thorough documentation of the database and its tables.
+PK-Sim uses an [SQLite](https://www.sqlite.org/index.html) database to store data models and templates. In this part of the developer documentation we will present this database with a thorough description of its structure and tables.
+
+More specifically, in the PK-Sim DB, we store:
+
+* Structural models for PBPK modeling of small and large molecules:
+* Anatomical and physiological parameters in different animal species and human populations for **healthy** individuals
+  * For details regarding parameterization of **diseased** individuals please refer to [tab_population_disease_states] (https://github.com/Open-Systems-Pharmacology/developer-docs/issues/19#tab_population_disease_states).

```

Comment:
remove the blank in the link between [] and ()

Reply:
done